app/javascript/dashboard/components/layout/AvailabilityStatus.vue

```diff
@@ -26,7 +26,9 @@
                 color-scheme="secondary"
                 class-names="status-change--dropdown-button"
                 :is-disabled="status.disabled"
-                @click="changeAvailabilityStatus(status.value)"
+                @click="
```

Comment:
the component doesn't depend on the current user anymore. But on current account id